src/Form.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import Input from "./Input";
+import { createNewConference } from "./api";
+
+const Form = () => {
+  const date = new Date();
+
+  const [state, setState] = useState({
+    name: "",
+    dateTime: date, // fix format of this to match BE
+    city: "",
+    description: "",
+    topic: ""
+  });
+
+  const handleChange = event => {
+    const value = event.target.value;
```

Comment:
You could destructure the event, so you wouldn't have to type event.target.x twice ;)